hathor/indexes/mempool_tips_index.py

```diff
@@ -27,22 +28,33 @@
     from hathor.transaction.storage import TransactionStorage
 
 SCOPE = Scope(
-    include_blocks=True,
+    include_blocks=False,
     include_txs=True,
     include_voided=True,
+    topological_order=False,
```

Comment:
Is this configuration just for the sake of initialization? I mean, this index must receive transaction in topological order to work properly (except during its initialization loop which will be executed only one time).

Reply:
The order is only used for initialization. Maybe it could be refactored to an `enum` called `InitializationOrder`, but I'd rather do that on a refactor PR.

Reply:
Ok. Thanks for the clarification.